### ISSUE TITLE
Fix accessibility issues in configuration UI.

### ISSUE THREAD

**Issue by schuppell** [SAP/spartacus#5152]
Configuration UI and Group navigation are not accessible.

**Comment by kacperknapik** [comment]
@schuppell please provide more info

**Comment by giancorderoortiz** [comment]
@developpeurweb could you please let us know if this is still an issue and if it can be fixed for 4.0?

**Comment by hackergil** [comment]
There is not enough info on this ticket to work on it.

**Comment by ChristophHi** [comment]
@Larisa-Staroverova: Could you update the ticket with the latest status on a11y? From what I understood, we are now compliant?

**Comment by Larisa-Staroverova** [comment]
After review with @developpeurweb (Miguel Estrada) no accessibility issues neither in the configuration nor in the overview pages were found. We got a recommendation from Miguel to make the group menu even more accessible that was done via #12089 in [PR](https://github.com/SAP/spartacus/pull/12348)
